rcl/src/rcl/publisher.c

```diff
@@ -197,6 +197,14 @@ rcl_publisher_init(
   goto cleanup;
 fail:
   if (publisher->impl) {
+    if (publisher->impl->rmw_handle) {
+      rmw_ret_t rmw_fail_ret = rmw_destroy_publisher(
+        rcl_node_get_rmw_handle(node), publisher->impl->rmw_handle);
+      if (RMW_RET_OK != rmw_fail_ret) {
+        RCUTILS_SAFE_FWRITE_TO_STDERR(rmw_get_error_string().str);
```

Comment:
The formatting wasn't required, see:

https://github.com/ros2/rcutils/blob/0c2cc271c3d3a357d16f15037606c15e6f7b0978/include/rcutils/error_handling.h#L284-L297

So if the other pr had that it may have been redundant?

Reply:
Yes, it's redundant. Good catch. Fixed in https://github.com/ros2/rcl/pull/834